src/components/DataElement/CitiesSplit.jsx

```diff
@@ -20,19 +23,31 @@ export default function CitiesSplit({ isLoading, data }) {
       >
         {isLoading && <Loader />}
         {data &&
-          <ResponsiveContainer height={data.length * 32} width={'99%'}>
-            <BarChart
-              data={data}
-              layout="vertical"
-            >
-              <YAxis dataKey="city" type="category" tick={{ fill: theme.colors.contentPrimary }} width={100} />
-              <XAxis type="number" hide />
-              <Tooltip
-                formatter={value => [value, 'Liczba']}
-              />
-              <Bar dataKey="count" fill={theme.colors.accent} />
-            </BarChart>
-          </ResponsiveContainer>}
+          <DataContext.Consumer>{({ setPosition, setZoom }) => (
+            <ResponsiveContainer height={data.length * 32} width={'99%'}>
+              <BarChart
+                data={data}
+                layout="vertical"
+              >
+                <YAxis
+                  dataKey="city"
+                  type="category"
+                  tick={{ fill: theme.colors.contentPrimary }}
```

Comment:
Do you think there's a way for ticks to have `cursor: pointer` ?

Reply:
I think yes, but to achieve this we probably should use customized tick Component. I'll try to implemet this as improvement ASAP :)

Reply:
Done, it was easier than expected. :D I tried adding cursor property directly to tick prop and it somehow worked.